go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/imdario/mergo v0.3.12 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.4.0 // indirect
```

Comment:
Let's added vendor rule to the Makefile.

```
vendor: $(GO)
        $(GO) mod tidy
        $(GO) mod vendor
```

Reply:
Why the go get is still needed?
Cannot you leave just the Makefile part in this commit and invoke `make vendor` on the next, where the NAD is used?